src/map.cpp

```diff
@@ -116,7 +116,7 @@ map::map(int mapsize)
 {
     nulter = t_null;
     my_MAPSIZE = mapsize;
-    grid.resize( my_MAPSIZE * my_MAPSIZE, nullptr );
+    grid.resize( my_MAPSIZE * my_MAPSIZE * OVERMAP_LAYERS, nullptr );
```

Comment:
Most likely we'd adjust _DEPTH or _HEIGHT as desired, so that would stay the same.  I could easily see deciding we need more of one or the other.